src/style.css

```diff
@@ -1 +1,92 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+:root {
+  --text-color: #3d4e61;
+  --input-box-shadow: 0 1px 8px 0 #3d4e611a;
+  --box-shadow: 0 3px 8px 0 #3d4e6133;
+  --linear-gradient: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+body {
+  /* the big search bar must have top indent 20px */
+  margin-top: 20px;
+}
+
+.form {
+  display: flex;
+  position: relative;
+  flex-direction: column;
+  gap: 20px;
+}
+
+.form__bar {
+  box-sizing: border-box;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  font-family: Avenir, Helvetica, Arial, sans-serif;
+  color: var(--text-color);
+  font-weight: 300;
+}
```

Comment:
Don't fix container size (if there is no such a requirement). By specifying a fixed height on `.form__bar--big` and `.form__bar--small`, you might run into issues with content overflow or unexpected layout behavior if the content exceeds the fixed size. Consider using `min-height` instead or ensure that the design can accommodate the fixed sizes.